src/visualization/maps/ReconMap/generateSubsytemsLayout.m

```diff
@@ -21,23 +21,24 @@
     for i = 1:length(cobra_model.rxns)
 
         if strcmp(cobra_model.subSystems(i), subsystem)
```

Comment:
as above. This needs to be adapted. The new
`isReactionInSubSystem` function would do this check.

Reply:
Sounds good, I will then make some updates on other functionality in the meantime.
Best
Alberto